packages/sdk-v2/src/sdk/claim/utils/getRecentClaims.ts

```diff
@@ -15,80 +17,105 @@ export const getRecentClaims = async (
   account: string,
   endpoints: string,
   library: any,
+  pools: Array<string>,
   withPools: boolean
 ): Promise<RecentClaims[] | undefined> => {
-  if (!library) return undefined;
-  const apiEndpoint = sample(endpoints.split(","));
-  const sender32 = ethers.utils.hexZeroPad(account, 32);
+  if (!library || !account) return undefined;
+  const { chainId } = await library.getNetwork();
+  const cacheKey = `GD_RecentClaims_${account}_${chainId}_${withPools}_${pools}`
```

Comment:
your prettier is not applying all the rules. cache-key should have `;` at the end

Reply:
fixed